oeps/oep-0014-proc-archive-repos.rst

```diff
@@ -246,3 +190,8 @@ Change History
 
 * Updated to use GitHub's archive capability.
 * Don't ask the community about public repos in the edx org that are not a part of Open edX.
+
+2020-03-20
+----------
+
+* Decide to use the new edx-unsupported org for all archived repos. Old way we were doing things is now recorded as Alternative 1: Archive In Place.
```

Comment:
From this comment, I'm reading that `archived == unsupported` and not necessarily `archived == unused` - is this the correct understanding?